gulpfile.babel.js

```diff
@@ -77,3 +77,12 @@ gulp.task('stop', finish => {
     }, 1000);
   });
 });
+
+
+gulp.task('platform-test', function (cb) {
+  child_process.exec('sh ./bootstrap/test/platform_test.sh', function (err, stdout, stderr) {
```

Comment:
IMHO, we should implement this test in javascript. (and please remove the test files..)
When the gulp command was executed, |sync_node && set_path_env| would already have been executed.
Please see |absolute.sh| in details.